settings.py

```diff
@@ -4,23 +4,6 @@
 import pkg_resources as res
```

Comment:
Is any of our parsers using data in the `settings.py` file still? I feel we should remove this file, it's quite confusing having a `settings.py` and a `configuration.yaml` too. This should go.

Reply:
I absolutely agree, but it is still being used in some parts of the code: e.g. the Gene2Phenotype parser. 
Added to the follow-up ticket.